content/en/news/releases/1.6.x/announcing-1.6/announcing-1.6.1/index.md

```diff
@@ -0,0 +1,49 @@
+- **Fixed** an issue where`istiod` fails to issue certificates to a remote cluster. `Istiod` now has support for the cluster name and certificate to generate the `injectionURL`. ([Issue 23879](https://github.com/istio/istio/issues/23879))
+- **Fixed** remote cluster's validation controller to check `istiod's` ready status endpoint ([Issue 23945](https://github.com/istio/istio/issues/23945))
+- **Improved** `regexp` fields validation to match Envoy's validation ([Issue 23436](https://github.com/istio/istio/issues/23436))
+- **Fixed** `istioctl analyze` to validate `networking.istio.io/v1beta1` resources ([Issue 24064](https://github.com/istio/istio/issues/24064))
+- **Fixed** typo of `istio` in `ControlZ` dashboard log ([Issue 24039](https://github.com/istio/istio/issues/24039))
+- **Fixed** tar name to directory translation ([Issue 23635](https://github.com/istio/istio/issues/23635))
+- **Improved** certificate management for multi-cluster and virtual machine setup from `samples/certs` directory to `install/tools/certs` directory
+- **Improved** `pilot-agent's` handling of client certificates when only a CA client certificate is present
+- **Improved** `istiocl upgrade` to direct users to the `istio.io` website to migrate from `v1alpha1` security policies to `v1beta1` security policies
+- **Fixed** release URL name for `istioctl upgrade`
+- **Fixed** `k8s.overlays` for cluster resources
+- **Fixed** `HTTP/HTTP2` conflict at Gateway ([Issue 24061](https://github.com/istio/istio/issues/24061) and [Issue 19690](https://github.com/istio/istio/issues/19690))
+- **Fixed** Istio Operator to respect the `--operatorNamespace` argument ([Issue 24073](https://github.com/istio/istio/issues/24073))
+- **Fixed** Istio operator hanging when uninstalling Istio ([Issue 24038](https://github.com/istio/istio/issues/24038))
+- **Fixed** TCP metadata exchange for upstream clusters that specify `http2_protocol_options` ([Issue 23907](https://github.com/istio/istio/issues/23907))
+- **Added** `sideEffects` field to `MutatingWebhookConfiguration` for `istio-sidecar-injector` ([Issue 23485](https://github.com/istio/istio/issues/23485))
+- **Improved** installation for replicated control planes ([Issue 23871](https://github.com/istio/istio/issues/23871))
+- **Fixed** `istioctl experimental` pre-check to report compatible versions of Kubernetes (1.14-1.18) ([Issue 24132](https://github.com/istio/istio/issues/24132))
+- **Fixed** Istio operator namespace mismatches causing a resource leak when pruning resources ([Issue 24222](https://github.com/istio/istio/issues/24222))
+- **Fixed** SDS Agent failing to start when Proxy uses file mounted certs for Gateways ([Issue 23646](https://github.com/istio/istio/issues/23646))
+- **Fixed** incorrect `ServiceAccount` being used in custom `IngressGateway` Deployment ([Issue 23303](https://github.com/istio/istio/issues/23303))
```

Comment:
I'd think this line can be removed as the same issue 23303 is referenced above already (as "handling of custom resource names for Gateways"). The original change did not land in any releases yet, and this specific fix is probably an implementation detail which is not too useful for public.
If it's better to over-communicate, I'd change "IngressGateway" to Gateway, as the fix was also applied to EgressGateway. (Sorry for the incorrect PR description!)

```suggestion
```

Reply:
Thanks for getting back to me. Removed.